src/explore-education-statistics-admin/src/components/editable/EditableContentBlock.tsx

```diff
@@ -195,10 +196,15 @@ const EditableContentBlock = ({
               ref={ref}
               onClick={isEditable ? onEditing : undefined}
             >
-              <ContentHtml
-                html={content || '<p>This section is empty</p>'}
-                sanitizeOptions={sanitizeOptions}
-              />
+              {/* Workaround to use inert, see https://github.com/facebook/react/issues/17157 */}
+              {/* eslint-disable-next-line react/jsx-props-no-spreading */}
+              <div {...{ inert: '' }}>
```

Comment:
It'd be better to do this via ambient declarations so we can have access to `inert` globally:


```ts
// types/react.d.ts

declare module 'react' {
  // eslint-disable-next-line @typescript-eslint/no-unused-vars
  interface HTMLAttributes<T> {
    inert?: '' | undefined;
  }
}

declare global {
  namespace JSX {
    interface IntrinsicAttributes {
      inert?: '' | undefined;
    }
  }
}

export {};
```

Also remember to re-export in `types.index.d.ts`:

```diff
export * from './govuk-frontend';
export * from './memoizee';
+ export * from './react';
export * from './react-app-env';
export * from './recharts';
export * from './util';
```

Reply:
ok, that's done now.